api/v1beta1/hyperconverged_types.go

```diff
@@ -446,20 +437,9 @@ type HyperConvergedFeatureGates struct {
 	// +default=false
 	DeployKubevirtIpamController *bool `json:"deployKubevirtIpamController,omitempty"`
 
-	// TODO: remove NonRoot in the next release
-
-	// Enables rootless virt-launcher.
-	//
-	// Deprecated: please use the root FG.
-	// +optional
-	// +kubebuilder:default=true
-	// +default=true
-	NonRoot *bool `json:"nonRoot,omitempty"`
-
 	// Disable mediated devices handling on KubeVirt
 	// +optional
-	// +kubebuilder:default=false
-	// +default=false
+	// Deprecated: there is no such FG in KubeVirt. This field is ignored
```

Comment:
i see there is such a feature gate in kubevirt:
https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-config/feature-gates.go#L53